### PR TITLE
Final edits to JOSS manuscript

### PR DESCRIPTION
**Pull request by joglekara**
PDF Plots

No inline integrals

Reference cleanup

Fokker-planck equation cleanup

### CODE REVIEW

**Comment by joglekara** [comment]
Addresses the PDF plots mentioned in https://github.com/openjournals/joss-reviews/issues/2182 and some other things

**Comment by joglekara** [comment]
@dpsanders @StanczakDominik @TomGoffrey , I just regenerated the PDF at the JOSS issue tracker. It has all the updates mentioned in this PR.

Looking forward to your thoughts

**Comment by joglekara** [comment]
@TomGoffrey, thanks for that feedback in https://github.com/openjournals/joss-reviews/issues/2182#issuecomment-655145700.  

I've made the necessary changes from points 1-4 here. 

Re point 5 -- LB operator not conserving momentum:

The LB operator is constructed to have a solution of driftless Maxwellian. In the original paper, the authors state that the operator conserves number density, provides diffusion, and yields "the Maxwell distribution" in the equilibrium state. They write down the following equilibrium distribution $f(v) = (2*pi*v0^2)^(3/2) * exp(-1/2*(v/v0)^2)$ (the driftless Maxwellian). 

In other words, they wanted a simplistic operator that would satisfy a few constraints, and this is what they got.

I think the DG operator was conceived to address this point directly, because it is a simple shift in the PDE that permits $f(v) \propto exp(-(v-v_d)^2/v0^2)$ as a solution, where $v_d$ is the drift velocity. 

I'll do some more searching too but I'd like to say these points are relatively well known.

**Comment by joglekara** [comment]
@TomGoffrey If it works for you, let's use this thread for the discussion on the normalization.

**Comment by TomGoffrey** [comment]
Agreed, this seems a reasonable place. My thought process is:

One ought to be able to start from the normalised equation, and recover the Vlasov equation (albeit multiplied by some constant)? Alternatively I could start from the Vlasov, define my normalisation and go from there, so here goes:

I'll ignore the normalisation of the distribution function for now, as it's a constant factor in all three terms of the Vlasov equation. Time derivative is simple:

df / dt = \omega df / d\tilde{t} (1), where \tilde{t} = t \omega, and \omega is the plasma frequency. 

Second term is only slightly more difficult:

v df / dx = \omega \tilde{v} df / d\tilde{x}, (2)

Here I'd defined \tilde{v} = v / v_t, and \tilde{x} = x \omega / v_t (= x / \lambda_D).

Up to this point I think we agree, after that however I think we diverge.

I define \tilde{E} = q_e \lambda_D E / (m_e v_t^2)= E q_e / (m_e v_t \omega), so:

q E / m df / dv = (q / m) (m_e / q_e) \omega \tilde{E} v_t df / dv =  (q / m) (m_e / q_e) \omega \tilde{E} df / d\tilde{v}

Finally I define \tilde{q} = q \ q_e, and \tilde{m} = m / m_e and I get:

\tilde{q} / \tilde{m} \tilde{E} \omega df / d\tilde{v}. (3)

Combining (1-3) and dropping the tilde notation I have:

\omega [ df/dt + v df/dx + qE/m df/dv] = 0.

Finally I restrict myself to electrons, such that q_e = m_e = 1 (note - this assumes I normalised by the electron charge), and I drop the omega term and I get:

df/dt + v df/dx + qE/m df/dv - your normalised Vlasov equation.

I hope this is an appropriate place for this discussion...

One ought to be able to start from the normalised equation, and recover the Vlasov equation (albeit multiplied by some constant)? Alternatively I could start from the Vlasov, define my normalisation and go from there, so here goes:

I'll ignore the normalisation of the distribution function for now, as it's a constant factor in all three terms. Time derivative is simple:

df / dt = \omega df / d\tilde{t} (1), where \tilde{t} = t \omega, and \omega is the plasma frequency. 

Second term is only slightly more difficult:

v df / dx = \omega \tilde{v} df / d\tilde{x}, (2)

Here I'd defined \tilde{v} = v / v_t, and \tilde{x} = x \omega / v_t (= x / \lambda_D).

Up to this point I think we agree, after that however I think we diverge.

I define \tilde{E} = q_e \lambda_D E / (m_e v_t^2)= E q_e / (m_e v_t \omega), so:

q E / m df / dv = (q / m) (m_e / q_e) \omega \tilde{E} v_t df / dv =  (q / m) (m_e / q_e) \omega \tilde{E} df / d\tilde{v}

Finally I define \tilde{q} = q \ q_e, and \tilde{m} = m / m_e and I get:

\tilde{q} / \tilde{m} \tilde{E} \omega df / d\tilde{v}. (3)

Combining (1-3) and dropping the tilde notation I have:

\omega [ df/dt + v df/dx + qE/m df/dv] = 0.

Finally I restrict myself to electrons, such that q_e = m_e = 1 (note - this assumes I normalised by the electron charge), and I drop the omega term and I get:

df/dt + v df/dx + qE/m df/dv - (your) normalised Vlasov equation.

Some comments/questions:

1. It's possible you've done something smarter than the above and I'm completely missing the point.
2. I didn't actually normalise f. I can make any choice and the above will be consistent, but it will impact the form of the normalised Poisson equation. I'll add a separate post assuming any of the above makes sense.
3. Is it worth considering changing the paper so \tilde{x} = x \lambda_D? Looks slightly neater than what is there now.
4. It's quite likely the above contains typos. There's only so much Latex I can parse in my head.

**Comment by TomGoffrey** [comment]
Part 2, normalisation of f.

To make the equation fully non-dimensional set \tilde{f} = f / f_0, where f_0 = n_e / v_T^3.

Now consider Poisson's equation:

\nabla E = \rho = q_e \int f d^3v.

LHS, use previous definitions of \tilde{E}, \tilde{x} to give:

m_e v_T^2 / (q_e \lambda_D^2) \tilde{\nabla} \tilde{E}.

RHS, using definitions of \tilde{q} and \tilde{v} from before and the above for \tilde{f}:

q \int f d^3v = q_e n_e \tilde{q} \int \tilde{f} d^3\tilde{v}.

Combine the two, divide through by q_e:

n_e [\tilde{\nabla} \tilde{E} - \tilde{q}\int\tilde{f}d^3\tilde{v}

Drop the n_e term, and the tilde notation:

\nabla E  = q \int f d^3v

Clearly to recover your form of the dimensionless Poisson equation q = -1, which suggests using the elementary charge as the normalisation factor. Based on the Poisson equation I think I prefer using the elementary charge as the normalisation factor, but this means \tilde{q} = -1, and the electric field term should be negative.

Of course one can rescale the distribution function by v_t^2 as you're 1D1P.

A perhaps foolish question - does the \rho_{net} = 1 - \rho_e imply that the charge on the ion is 1? If so, might warrant a comment? If not, well, it's gone midnight, please ignore me.

**Comment by joglekara** [comment]
@TomGoffrey , 

I think I agree with you overall. This is where I agree:
Generalized Vlasov Equation for species `s` is
`df/dt + v df/dx + q_s/m_s E df/dv = 0. `
`dE/dx = \sum rho_s = \sum q_s n_s`

lets normalize `q_s` by `q_0=+1.6e-19` and `m_s` by `m_0=9.11e-31`

For electrons, assuming `q_e = -1  * q_0`, and `m_e = 1 * m_0`:
`df/dt + v df/dx - E df/dv = 0.`

The minus sign here is important. 

For Poisson, where `q_i = Z * q_0`, and quasineutrality needs `Z n_i ~ n_e`
`dE/dx = \sum rho = q_i n_i + q_e n_e = Z n_i - n_e`
Assuming `n_i ~ constant`
`dE/dx = 1 - int f dv`

The key concept is that `E` is a global field that is the outcome of the total charge. As for `Z`, I think `Z` gets normalized out in the Poisson equation so I don't think there's a strict `Z=1` requirement.

**Comment by TomGoffrey** [comment]
Agreed on the charge state and final form of the Vlasov equation.

In the description of the normalisation it would be clearer to add \tilde{q} = q / e, and \tilde{m} = m / m_e, and as discussed above I think the given definitions of \tilde{E} and \tilde{f} are incorrect.

**Comment by joglekara** [comment]
Thanks. Great catches, all. The changes have been pushed

**Comment by joglekara** [comment]
@TomGoffrey I think these are good to merge in but would love your approval on this PR if you have a moment!

**Comment by TomGoffrey** [comment]
@joglekara I'm so sorry! I got the email notifying me that you'd merged master into paper, but I misread it as paper being merged into master, so thought all was done. I'll approve now!

**Comment by joglekara** [comment]
no worries, thanks! merging now.
